main/mainworker.cpp

```diff
@@ -10095,6 +10095,7 @@ void MainWorker::decode_General(const CDomoticzHardwareBase* pHardware, const tR
 		(subType == sTypePercentage) ||
 		(subType == sTypeWaterflow) ||
 		(subType == sTypePressure) ||
+		(subType == sTypeVisibility) ||
```

Comment:
This change will cause other hardware to have new ID's, please undo

Reply:
For the Visibility sensor the NodeID is also not using the ChildID, sort of same as for Percentage. Maybe for consistency somehow make these sensors behave the same?

Will revert for this PR.